trigger/tests/test_handler.py

```diff
@@ -116,12 +116,6 @@ def test_error_path(self, mock_esm):
         calls = [call_0, call_1]
         mock_esm.assert_has_calls(calls, any_order=True)
 
-    @mock.patch.dict('os.environ', {'STATE_MACHINE_ARN': state_machine_arn, 'AWS_DEPLOYMENT_REGION': region, 'OBJECT_SIZE_LIMIT': str(10**7)})
-    @mock.patch('trigger.handler.execute_state_machine', autospec=True)
-    def test_big_files_do_not_trigger_step_function(self, mock_esm):
-        lambda_handler(self.sqs_event_big_s3_file, self.context)
```

Comment:
I don't believe `self.sqs_event_big_s3_file` has any other usages beyond this one here. It can be removed along with this test.